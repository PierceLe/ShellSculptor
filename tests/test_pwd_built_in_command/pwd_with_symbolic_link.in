cd ~/test_data/symbolic_link_detect
pwd
