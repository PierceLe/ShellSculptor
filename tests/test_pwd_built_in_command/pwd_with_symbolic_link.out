>> cd ~/test_data/symbolic_link_detect
>> pwd
/home/test_data/symbolic_link_detect

